doc/neomake.txt

```diff
@@ -31,13 +31,13 @@ fantastic vim plugins such as syntastic and dispatch.
 ==============================================================================
 2. Commands                                                   *neomake-commands*
 
-                                                         *:Neomake* *:NeomakeFile*
+                                                       *:Neomake* *:NeomakeFile*
```

Comment:
I turned it off in my vimrc, I'll leave it up to you since it's just a matter of personal style